executor/aggfuncs/func_count.go

```diff
@@ -391,5 +398,6 @@ func (e *countPartial) UpdatePartialResult(sctx sessionctx.Context, rowsInGroup
 func (*countPartial) MergePartialResult(sctx sessionctx.Context, src, dst PartialResult) (memDelta int64, err error) {
 	p1, p2 := (*partialResult4Count)(src), (*partialResult4Count)(dst)
 	*p2 += *p1
-	return 0, nil
+	memDelta = DefPartialResult4CountSize * 2
```

Comment:
memory usage does not change after the merge, `return 0`, nil here

Reply:
okay, Sorry for this repeated mistake. 
pointer assume cost 0 and no struct initialization in this function.